products/health-care/digital-health-modernization/product/roadmap.md

```diff
@@ -1,9 +1,15 @@
 # DRAFT Digital Health Modernization Product Roadmap
-This is a rough draft of a Digital Health Modernization roadmap. It has no buy-in from teams who would be affected. It exists as a conversation-starter. Assumes [Product Strategy A](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/product/README.md).
+This is a rough draft of a Digital Health Modernization roadmap. It has no buy-in from teams who would be affected. It exists as a conversation-starter. Assumes [Product Strategy A](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/product/README.md). It assumes My HealtheVet will continue to operate in parallel during the bulk of this work.
+
+# Vision
+- Paraphrase: A single, high-quality UX for each Veteran-facing tool. 
+
+# Mission
+- Create a viable, comprehensive patient portal on VA.gov before Cerner de facto usurps running VA's patient portal. 
 
 # Roadmap
 ## Now
-Items in Now are not strongly order-dependent. 
+Items in Now are not strongly order-dependent. This describes how we might agile-ly add core clinical affordances to VA.gov. 
 
 ### Optimize a single VA.gov health page (ex., prescriptions) for call to action over content.
```

Comment:
If it's not out of bounds, I'd like to include work on MHV to improve the redirects, too (this applies only to health functionality that exists on VA.gov, so for now, just VAOS).